templates/style/base.scss

```diff
@@ -25,12 +75,68 @@ body {
     padding: 0;
     margin: 0;
 
+    * {
+        -webkit-box-sizing: border-box;
+        -moz-box-sizing: border-box;
+        box-sizing: border-box;
+    }
+
+    h1 {
+        font-size: 1.5em;
+    }
+    h1, h2, h3, h4 {
+        font-family: "Fira Sans",sans-serif;
+    }
+    h1, h2,
+    h3:not(.impl):not(.method):not(.type):not(.tymethod):not(.notable),
+    h4:not(.method):not(.type):not(.tymethod):not(.associatedconstant) {
```

Comment:
We shouldn't need all these special cases since we don't use these classes, do we? (same below)

Reply:
Indeed! Forgot to remove all the rules...